src/_pytest/assertion/util.py

```diff
@@ -308,6 +308,22 @@ def _diff_text(left: str, right: str, verbose: int = 0) -> list[str]:
                 ]
                 left = left[:-i]
                 right = right[:-i]
+        shortest = min(left, right, key=lambda x: len(x))
+        lines = j = 0
+        if shortest.count("\n") >= 8:
```

Comment:
I don't think this should be hardcoded, there's an option to configure this as far as a I remember.

Reply:
As far as I was able to find, `truncate_if_required` always calls `_truncate_explanation` in a way that `DEFAULT_MAX_LINES` (8 lines) and `DEFAULT_MAX_CHARS` (640 characters) are used as the limit:

https://github.com/pytest-dev/pytest/blob/a7d5a8eba9addd119432fa71880b51052a89812f/src/_pytest/assertion/truncate.py#L19-L25

https://github.com/pytest-dev/pytest/blob/a7d5a8eba9addd119432fa71880b51052a89812f/src/_pytest/assertion/truncate.py#L34-L48

While not appearing to be easily configurable, a user can still monkeypatch e.g. `truncate.DEFAULT_MAX_CHARS` to modify the behavior. So I changed it to import these constants from truncate.py instead.

Does that address it?

Reply:
Sure, sorry it seem I was remembering wrong.